Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -240,6 +249,51 @@ class UserScriptManager {
         }
         return WKUserScript(source: source, injectionTime: .atDocumentStart, forMainFrameOnly: false)
     }()
+    
+    private let PlaylistSwizzlerScript: WKUserScript? = {
+        guard let path = Bundle.main.path(forResource: "PlaylistSwizzler", ofType: "js"),
+              let source = try? String(contentsOfFile: path) else {
+            log.error("Failed to load PlaylistSwizzler.js")
+            return nil
+        }
+        //return WKUserScript(source: source, injectionTime: .atDocumentStart, forMainFrameOnly: false)
+        return WKUserScript.createInDefaultContentWorld(source: source, injectionTime: .atDocumentStart, forMainFrameOnly: false)
+    }()
+    
+    private let PlaylistHelperScript: WKUserScript? = {
```

Comment:
What do you think about gating the script injection for `Playlist` scripts by `isPlaylistSupportedSiteURL`?

Reply:
Can't change it because you can still add to playlist even if the URL is unsupported, by going to the "Share" menu and hitting add to playlist for an item that may not be in the supported list.

The current list is like: "vimeo", "dailymotion", "youtube", "twitch", etc.. it's pretty small. But there are some websites that have video that can still be added even when it isn't a supported site, via the share menu.